client/src/app/components/filters/filters.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { StateService } from 'src/app/services/state.service';
+
+@Component({
+  selector: 'app-filters',
+  templateUrl: './filters.component.html',
+  styleUrls: ['./filters.component.scss'],
+})
+export class FiltersComponent implements OnInit {
+  public topics = ['people', 'cash', 'data', 'all'];
+  public state: StateService;
+  public tagCount: any;
+
+  constructor(state: StateService) {
+    this.state = state;
+  }
```

Comment:
```suggestion
  public tagCount: any;

  constructor(public state: StateService) {}
```

I think this should be enough, as the mapping to `this.*` happens automatically...